src/data/markdown/docs/03 cloud/04 Project and Team Management/04 Azure AD SAML SSO.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 'Azure AD SAML SSO'
+excerpt: 'Guide on setting up Azure AD to act as a SAML SSO IdP with k6 Cloud'
+---
+
```

Comment:
Suggest adding an intro about the feature. What is the feature and the benefits in the app?